xradar/georeference/transforms.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+# Copyright (c) 2022, openradar developers.
+# Distributed under the MIT License. See LICENSE for more info.
+
+import numpy as np
+
+
+def antenna_to_cartesian(
+    ranges, azimuths, elevations, earth_radius=6371000, effective_radius_fraction=None
+):
+    """
+    Return Cartesian coordinates from antenna coordinates.
+    Parameters
+    ----------
+    ranges : array
+        Distances to the center of the radar gates (bins) in kilometers.
+    azimuths : array
+        Azimuth angle of the radar in degrees.
+    elevations : array
+        Elevation angle of the radar in degrees.
+    earth_radius: float
+        Radius of the earth (default is 6371000 m).
+    effective_radius_fraction: float
+        Fraction of earth to use for the effective radius (default is 4/3).
+    Returns
+    -------
+    x, y, z : array
+        Cartesian coordinates in meters from the radar.
+    Notes
+    -----
+    The calculation for Cartesian coordinate is adapted from equations
+    2.28(b) and 2.28(c) of Doviak and Zrnic [1]_ assuming a
+    standard atmosphere (4/3 Earth's radius model).
+    .. math::
+        z = \\sqrt{r^2+R^2+2*r*R*sin(\\theta_e)} - R
+        s = R * arcsin(\\frac{r*cos(\\theta_e)}{R+z})
+        x = s * sin(\\theta_a)
+        y = s * cos(\\theta_a)
+    Where r is the distance from the radar to the center of the gate,
+    :math:`\\theta_a` is the azimuth angle, :math:`\\theta_e` is the
+    elevation angle, s is the arc length, and R is the effective radius
+    of the earth, taken to be 4/3 the mean radius of earth (6371 km).
+    References
+    ----------
+    .. [1] Doviak and Zrnic, Doppler Radar and Weather Observations, Second
+        Edition, 1993, p. 21.
+    """
+    if effective_radius_fraction is None:
+        effective_radius_fraction = 4.0 / 3.0
+    theta_e = np.deg2rad(elevations)  # elevation angle in radians.
```

Comment:
Just a personal style thing, I'd prefer comments above. But no strict necessity.

Reply:
Me too - I'll correct that